Assets/MixedRealityToolkit/Inspectors/MixedRealityStandardShaderGUI.cs

```diff
@@ -593,9 +593,9 @@ protected void FluentOptions(MaterialEditor materialEditor, Material material)
                     materialEditor.ShaderProperty(borderLightUsesHoverColor, Styles.borderLightUsesHoverColor, 2);
                 }
 
-                if (mode == RenderingMode.TransparentCutout || mode == RenderingMode.Transparent ||
-                    (mode == RenderingMode.Custom && customMode == CustomRenderingMode.TransparentCutout) ||
-                    (mode == RenderingMode.Custom && customMode == CustomRenderingMode.Transparent))
+                if (mode == RenderingMode.Cutout || mode == RenderingMode.Fade || mode == RenderingMode.Transparent ||
```

Comment:
It seems like this is adding a new case to this `if` (there were two on this line and now there are three):

```suggestion
                if (mode == RenderingMode.Cutout || mode == RenderingMode.Fade ||
```

Reply:
Yeah, I found that little bug when looking over the rename. It should be all 3 conditionals (not just the 2). But, thanks for double checking!